python/lsst/ts/standardscripts/base_focus_sweep.py

```diff
@@ -120,6 +120,12 @@ def get_schema(cls) -> dict:
                 description: Number of steps to take inside the focus window.
                 type: number
                 minimum: 3
+              focus_steps:
+                description: Steps to take for the focus sweep.
```

Comment:
We can then update this description to say "User-provided sequence of focus steps to take for the focus sweep, used for unevenly spaced steps."